components/isceobj/Util/ImageUtil/ImageLib.py

```diff
@@ -67,8 +67,8 @@
 
 #######Current list of constants
 constDict = { "PI"  : np.pi,
-              "J"   : np.complex(0.0, 1.0),
-              "I"   : np.complex(0.0, 1.0),
+              "J"   : np.complex64(0.0, 1.0),
+              "I"   : np.complex64(0.0, 1.0),
```

Comment:
```suggestion
              "J"   : 1j,
              "I"   : 1j,
```

Alternatively, can we just use python complex literals, or do these need to be numpy types?

Reply:
@rtburns-jpl I think python complex literals should work